packages/schema/src/idl.rs

```diff
@@ -74,7 +74,7 @@ pub struct JsonApi<'v> {
     query: RootSchema,
     migrate: Option<RootSchema>,
     sudo: Option<RootSchema>,
-    responses: BTreeMap<String, RootSchema>,
+    responses: BTreeMap<&'static str, RootSchema>,
```

Comment:
Can we keep this owned String for simplicity? If the key is created with the new macro, it should not make a difference for the user.

Reply:
The map's key type is still `&'static str`